compiler/sem/injectdestructors.nim

```diff
@@ -595,8 +581,8 @@ proc processScope(c: var Con; s: var Scope; ret: PNode): PNode =
   if s.vars.len > 0:
     let varSection = newNodeI(nkVarSection, ret.info)
     for tmp in s.vars:
-      varSection.add newTree(nkIdentDefs, newSymNode(tmp), newNodeI(nkEmpty, ret.info),
-                                                           newNodeI(nkEmpty, ret.info))
+      varSection.add newTree(nkIdentDefs,
+        [newSymNode(tmp), newNodeI(nkEmpty, ret.info), newNodeI(nkEmpty, ret.info)])
```

Comment:
```suggestion
      varSection.add nkIdentDefs.newTree:
        [newSymNode(tmp), newNodeI(nkEmpty, ret.info), newNodeI(nkEmpty, ret.info)]
```

Reply:
This can't work because the `:` binds to `add`.

Reply:
Oh yea, forgot about it, here's CPS style:

```suggestion
      varSection.add:
        nkIdentDefs.newTree:
          [newSymNode(tmp), newNodeI(nkEmpty, ret.info), newNodeI(nkEmpty, ret.info)]
```